cases/transactions.test.js

```diff
@@ -0,0 +1,251 @@
+    getTransactionSchema
+} from "./util/schema";
+
+jest.setTimeout(60000);
+
+const urlBuilder = new URL(process.env.DOMAIN);
+const domain = urlBuilder.toString();
+const secret = "SAUOSXXF7ZDO5PKHRFR445DRKZ66Q5HIM2HIPQGWBTUKJZQAOP3VGH3L";
+const keyPair = StellarSDK.Keypair.fromSecret(secret);
+
+describe("Transactions", () => {
+    let toml;
+    let enabledCurrency;
+    let jwt;
+
+    beforeAll(async () => {
+        toml = await getTomlFile(domain);
+        jwt = await getSep10Token(domain, keyPair);
+
+        const infoResponse = await fetch(toml.TRANSFER_SERVER + "/info", {
+            headers: {
+                Origin: "https://www.website.com"
+            }
+        });
+
+        const infoJSON = await infoResponse.json();
+        const currencies = Object.keys(infoJSON.deposit);
+
+        enabledCurrency = currencies.find(
+            currency => infoJSON.deposit[currency].enabled
+        );
+
+        expect(toml.TRANSFER_SERVER).toBeDefined();
+        expect(toml.WEB_AUTH_ENDPOINT).toBeDefined();
+    });
+
+    it("has CORS on the transactions endpoint", async () => {
+        const response = await fetch(toml.TRANSFER_SERVER + "/fee", {
+            headers: {
+                Origin: "https://www.website.com"
+            }
+        });
+        expect(response.headers.get("access-control-allow-origin")).toBe("*");
+    });
+
+    it("return proper formatted transactions list", async () => {
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, true);
+
+        const response = await fetch(
+            toml.TRANSFER_SERVER + `/transactions?asset_code=${enabledCurrency}`, {
+                headers: {
+                    Authorization: `Bearer ${jwt}`
+                }
+            }
+        );
+
+        const json = await response.json();
+        expect(response.status).toEqual(200);
+        expect(json.error).not.toBeDefined();
+        expect(json).toMatchSchema(transactionsSchema);
+
+        json.transactions.forEach((transaction) => {
+            const isDeposit = transaction.kind === 'deposit';
+            const schema = getTransactionSchema(isDeposit);
+            expect(transaction).toMatchSchema(schema.properties.transaction);
+        });
+    });
+
+    it("return proper amount of transactions with limit param", async () => {
+        await createTransaction(enabledCurrency, keyPair.publicKey(), toml, jwt, true, false);
+
+        const response = await fetch(
+            toml.TRANSFER_SERVER + `/transactions?asset_code=${enabledCurrency}&limit=1`, {
+                headers: {
+                    Authorization: `Bearer ${jwt}`
+                }
+            }
+        );
+
+        const json = await response.json();
+        expect(response.status).toEqual(200);
+        expect(json.error).not.toBeDefined();
+        expect(json.transactions.length).toBe(1);
+    });
+
+    it("return proper transactions with no_older_than param", async () => {
```

Comment:
This test doesn't check that there were any transactions returned. There should be at least 2.